model/jpa/src/main/resources/META-INF/jpa-changelog-1.0.0.Final.xml

```diff
@@ -17,8 +17,11 @@
   -->
 
 <databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.2.xsd">
-    <changeSet author="sthorger@redhat.com" id="1.0.0.Final">
-        <preConditions onSqlOutput="TEST" onFail="MARK_RAN"> 
+    <changeSet author="sthorger@redhat.com" id="1.0.0.Final-KEYCLOAK-5461">
+        <preConditions onSqlOutput="TEST" onFail="MARK_RAN">
+            <not>
+                <changeSetExecuted id="1.0.0.Final-KEYCLOAK-5461" author="sthorger@redhat.com" changeLogFile="META-INF/jpa-changelog-1.0.0.Final.xml"/>
```

Comment:
@martin-kanis @hmlnarik I've re-run the tests and now I get liquibase.exception.MigrationFailedException: Migration failed for change set META-INF/jpa-changelog-1.0.0.Final.xml::1.0.0.Final-KEYCLOAK-5461::sthorger@redhat.com:
...
...
...
Caused by: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Table 'client' already exists

The problem is IMO in the change of changelog IDs on this file (the same for other changes). This precondition has to assume that changese id="1.0.0.Final" was not executed.